{{cookiecutter.app_name}}/database/migrations/20210223141838_create_healths_table.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE healths (
```

Comment:
This table does not do much and we would need to delete it. While I understand the intention was to demonstrate it works, the template should be kept as clean as possible (i.e. nothing needs to be deleted). Either we remove it or we can keep it as part of the template but when creating a new project, there should be a command to remove all "test data" e.g. `make prune-template`.

Reply:
I think it is better to remove it to avoid confusion and we can add some testing steps as a part of the pull request.

Reply:
Removed migration file in a93a1e0b1ec2e6ddac7befc2a7909831d59fa0c9 and added testing step at the Insight section also updated README for the migration command usage in 0dad0345b6e2da4b2a1c2fcdc27ade30eb98a76d